WrathCombo/Combos/PvP/GNBPVP.cs

```diff
@@ -47,19 +48,29 @@ internal const ushort
                 ReadyToRaze = 4293;
 
         }
+        public class Config
+        {
+            public const string
+                corundumThreshold = nameof(corundumThreshold);
 
+        }
         internal class GNBPvP_Burst : CustomCombo
         {
             protected internal override CustomComboPreset Preset { get; } = CustomComboPreset.GNBPvP_Burst;
 
             float GCD = GetCooldown(KeenEdge).CooldownTotal; // 2.4 base in PvP
-            bool enemyGuard = TargetHasEffect(PvPCommon.Buffs.Guard); //Guard check
-            bool inGF = JustUsed(GnashingFang, 3f) || JustUsed(SavageClaw, 3f) || JustUsed(WickedTalon, 2f);
-
+            
             protected override uint Invoke(uint actionID, uint lastComboMove, float comboTime, byte level)
             {
                 if (actionID is KeenEdge or BrutalShell or SolidBarrel)
                 {
+                    int corundumThreshold = GetOptionValue(Config.corundumThreshold);
+                    bool enemyGuard = TargetHasEffect(PvPCommon.Buffs.Guard); //Guard check
+                    bool inGF = JustUsed(GnashingFang, 3f) || JustUsed(SavageClaw, 3f) || JustUsed(WickedTalon, 2f);
+
+                    if (CanWeave(ActionWatching.LastWeaponskill) && IsEnabled(CustomComboPreset.GNBPvP_Corundum) && PlayerHealthPercentageHp() <= corundumThreshold && IsOffCooldown(HeartOfCorundum))
```

Comment:
Looks functional, but I would highly advise against using `canWeave` here (you're just throwing away free damage from Continuation procs). Let them go through even if it's not in a weave window.
PvP is far too chaotic, and unlike PvE, you cannot guarantee weaves.

Reply:
Happened to look at a different block at the same time. This is fine.